Extension/package.json

```diff
@@ -1032,6 +1032,7 @@
     "@types/mocha": "^2.2.43"
   },
   "dependencies": {
+    "jsonc-parser": "^1.0.0",
```

Comment:
@sean-mcmanus mentioned that we need to file an OSS request for this and have LCA add it to our ThirdPartyNotices.txt